app/models/user.rb

```diff
@@ -102,13 +102,9 @@ def self.from_omniauth(access_token)
   # (e.g. assigned to current line and in user.patients)
 
   def recently_called_patients(lines = LINES)
-    patients.in(_line: lines)
-            .select { |patient| recently_called_by_user? patient }
-  end
-
-  def call_list_patients(lines = LINES)
-    patients.in(_line: lines)
-            .reject { |patient| recently_called_by_user? patient }
+    Patient.in(_line: lines, user_ids: BSON::ObjectId(id)).where( :$and =>
```

Comment:
I think the inclination here is really powerful (filtering on the db level, rather than the others)! I think that's a great insight.

I'd like to confirm that the number of db calls here reduces. I think when we wrote this we were trying to take advantage of some result caching by executing basically the same query twice (`patients.in(_line: lines)`), but that didn't quite pan out. Can we compare logs from the rails console in this PR to confirm stuff is cooking the way we want it to?